README.md

```diff
@@ -187,7 +250,41 @@ Next, generate a Personal Access Token for bootstrapping the `core` packages, as
 
-## Bootstrap `core` packages
+## Bootstrap Builder with Habitat Packages (**New**)
+
+Chef Habitat Builder on-prem has no pre-installed package sets. You must populate your Builder instance by uploading packages.
+With Habitat *0.88.0*, two new commands were introduced to assist in bootstrapping an on-prem Builder instance with a set of stable packages:
+
+1. *hab pkg download*
+1. *hab pkg bulkupload*
+
+As you can see from the commands above, the package Bootstrap flow is comprised of two main phases: a download from the public [SaaS Builder](https://bldr.habitat.sh) followed by a bulkupload to your on-prem Builder instance(s). Historically, we bootstraped on-prem-builders by downloading all the packages in 'core' for all targets. That amounted to ~15GB and was both too much and too little, in that many of the packages weren't needed, and for many patterns (effortless) other origins were needed.
+
+The [new bootstrap process flow](https://forums.habitat.sh/t/populating-chef-habitat-builder-on-prem/1228) allows you to easily customize your Bootstrap package set or use pre-populated [Package Seed Lists](https://github.com/habitat-sh/on-prem-builder/blob/master/package_seed_lists/README.md) files.
+
+The following section illustrates the steps required to bootstrap the on-prem Builder with the [Effortless Linux](https://github.com/habitat-sh/on-prem-builder/blob/master/package_seed_lists/effortless_x86_64-linux_stable) package seed list. Simply repeat the following download/bulkupload flow for any other package seed lists you think you will need to have in your on-prem Builder, or even create your own custom package seed list file:
+
+1. Phase 1: download
+
+    ```bash
+    export HAB_AUTH_TOKEN=<your_public_Builder_instance_token>
+    cd on-prem-builder
+    hab pkg download --target x86_64-linux --channel stable --file package_seed_lists/effortless_x86_64-linux_stable --download-directory builder_bootstrap
+    ```
+    > Note: If the on-prem Builder is Airgapped, this phase must be completed on a system with Internet connectivity. The downloaded content will need to be zipped and then transferred to the Builder system for phase 2.
+
+1. Phase 2: bulkupload
+
+     **Important**: Inspect the contents of the `builder_bootstrap/artifacts` directory created from the download command above. For each of the origins (`core`, `effortless`, etc),  create the origin name if one doesn't exist already in the on-prem Builder UI before starting the bulkupload.
+
+    ```bash
+    export HAB_AUTH_TOKEN=<your_on-prem_Builder_instance_token>
+    hab pkg bulkupload --url https://your-builder.tld --channel stable builder_bootstrap/
+    ```
+
+## Bootstrap `core` packages (**Deprecated**)
+
+*Important*: This methodology is now deprecated in favor of the download/bulkupload flow described above.
```

Comment:
Also add the deprecation notice line to the "Synchronizing" section below?

Reply:
Agreed. Added in https://github.com/habitat-sh/on-prem-builder/pull/212/commits/bf14ce0f10ff40c35f15c48f6122f570929a452d